main.tf

```diff
@@ -21,11 +21,16 @@
  *****************************************/
 data "google_compute_zones" "available" {
   count = local.zone_count == 0 ? 1 : 0
+}
 
-  provider = google
+provider "google" {
+  project = "rare-style-447407-s1"
```

Comment:
Hi @samuelarogbonlo - I suspect these and the later files were unintentionally included.